packages/ui/src/components/Switch/shared.ts

```diff
@@ -12,19 +12,36 @@ export interface CustomSwitchProps extends Omit<SwitchBaseProps, 'classes'> {
  */
 export type SwitchClasses = {
```

Comment:
Could we reimplement this type using type generation based on `SwitchSize`? This will make its type declaration way more stable and have no need to change in case new size was added